client/my-sites/domains/domain-management/gsuite/gsuite-purchase-cta/features.jsx

```diff
@@ -0,0 +1,163 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import PropTypes from 'prop-types';
+import React, { Fragment } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import { ADDING_GOOGLE_APPS_TO_YOUR_SITE } from 'lib/url/support';
+import { composeAnalytics, recordGoogleEvent, recordTracksEvent } from 'state/analytics/actions';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+class GSuitePurchaseCtaFeatures extends React.Component {
+	handleLearnMoreClick = () => {
+		this.props.learnMoreClick( this.props.domainName );
+	};
+
+	getStorageText() {
+		const { productSlug, translate } = this.props;
+		if ( 'gapps' === productSlug ) {
+			return translate( 'Get 30GB of storage for all your files synced across devices.' );
+		} else if ( 'gappsbusiness' === productSlug ) {
+			return translate( 'Get 30GB of storage for all your files synced across devices.' );
+		}
+	}
+
+	render() {
+		const { domainName, type, translate } = this.props;
+
+		return (
+			<Fragment>
+				<div
+					className={
+						'grid' === type
+							? 'gsuite-purchase-cta__add-google-apps-card-features-grid'
+							: 'gsuite-purchase-cta__add-google-apps-card-features-list'
+					}
+				>
+					<div className="gsuite-purchase-cta__add-google-apps-card-feature">
+						<div className="gsuite-purchase-cta__add-google-apps-card-feature-block">
+							<img alt="Gmail Logo" src="/calypso/images/g-suite/logo_gmail_48dp.svg" />
+						</div>
+						<div className="gsuite-purchase-cta__add-google-apps-card-feature-block">
+							<h5 className="gsuite-purchase-cta__add-google-apps-card-feature-header">
+								{ translate( 'Gmail for @%(domain)s', {
```

Comment:
This looks a bit weird on my Chromebook, and is missing the gmail icon. The @domainname looks like an error, as if it's missing the username portion of the account. Maybe drop the @? Or if there's only one address, add it in?

![screenshot 2019-02-25 at 8 18 26 pm](https://user-images.githubusercontent.com/51896/53387341-52695800-393b-11e9-9420-e3ffb177d6f4.png)